src/BestBooks.js

```diff
@@ -9,6 +11,37 @@ class BestBooks extends React.Component {
   }
 
   /* TODO: Make a GET request to your API to fetch books for the logged in user  */
+  componentDidMount() {
+    this.fetchBooks();
+  }
+
+  async fetchBooks(email = null){
+    let apiURL = `${process.env.REACT_APP_SERVER}/books`;
+
+    console.log(apiURL);
+
+    if(email){
+      apiURL += `?email=${email}`
+    }
+
+    try{
+      const response = await axios.get(apiURL);
+
+        console.log(response.data)
+
+      this.setState({books: response.data});
+    
+    }catch(error){
+      console.log(error);
+    }
+  }
```

Comment:
Nice work so far! Clean up spacing here before next merge to keep code readable/clean.